Makefile

```diff
@@ -126,9 +126,6 @@ test-binaries: test/bin2img/bin2img test/copyimg/copyimg test/checkseccomp/check
 MANPAGES_MD := $(wildcard docs/*.md)
 MANPAGES    := $(MANPAGES_MD:%.md=%)
 
-docs/%.1: docs/%.1.md .gopathok
-	(go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@) || ($(GOPATH)/bin/go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@)
-
```

Comment:
The double-command with ``||`` kicks off my OCD big-time...and then it's repeated a few more times in other targets as well.

Can't the command get set in stone, then re-used just once in each of the targets?

For example:

```
--- a/Makefile
+++ b/Makefile
@@ -125,15 +125,16 @@ test-binaries: test/bin2img/bin2img test/copyimg/copyimg test/checkseccomp/check
 
 MANPAGES_MD := $(wildcard docs/*.md)
 MANPAGES    := $(MANPAGES_MD:%.md=%)
+GOMD2MAN    := $(shell type -P go-md2man || type -P $(GOPATH)/bin/go-md2man)

 docs/%.5: docs/%.5.md .gopathok
-       (go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@) || ($(GOPATH)/bin/go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@)
+       ${GOMD2MAN} -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
 
 docs/%.8: docs/%.8.md .gopathok
-       (go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@) || ($(GOPATH)/bin/go-md2man -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@)
+       ${GOMD2MAN} -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
 
 docs: $(MANPAGES)
```

?

Reply:
ahh okay, yes, that's a strategy I need to get use to 🤕    I've got a (really) bad habbit of just jamming all the changes together, fixes + features.  bad-bad-ju-ju